core/src/main/java/com/homeaway/streamplatform/streamregistry/model/StreamClient.java

```diff
@@ -16,23 +16,43 @@
 package com.homeaway.streamplatform.streamregistry.model;
 
 import java.util.List;
+import java.util.Objects;
 
 import javax.validation.constraints.NotNull;
 
+import lombok.AllArgsConstructor;
 import lombok.Data;
 
+@AllArgsConstructor
 @Data
 public class StreamClient {
 
     /**
      * Name of the producer or Consumer
      */
     @NotNull
-    final String name;
+    String name;
```

Comment:
This is in-line with the current master impl ( i.e. we just merged this to avoid and conflict ). I see the default scopes assigned to other models in the same package ( https://github.com/homeaway/stream-registry/tree/master/core/src/main/java/com/homeaway/streamplatform/streamregistry/model ). My take is to convert this ( and all other model members ) to `private`